AlphaWallet/Settings/Views/ServerImageTableViewCell.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  ServerImageTableViewCell.swift
+//  AlphaWallet
+//
+//  Created by Jerome Chan on 18/4/22.
+//
+
+import UIKit
+
+class ServerImageTableViewCell: UITableViewCell {
+
+    // MARK: - Properties
+
+    // MARK: Private
+    private var chainIconView: ImageView = ImageView()
```

Comment:
Some of these can be `let`s

Reply:
Better to always default to `let` unless it's not possible.

Reply:
Actually, all of them can be lets...